cmd/clef/main.go

```diff
@@ -814,16 +814,6 @@ func addPluginAPIs(pm *plugin.PluginManager, rpcAPI []rpc.API, ui core.UIClientA
 	return append(rpcAPI, approvalPluginAPIs...)
 }
 
-// splitAndTrim splits input separated by a comma
-// and trims excessive white space from the substrings.
-func splitAndTrim(input string) []string {
```

Comment:
this was removed on geth?

Reply:
yes, in a conflit merge I prefered to let it because I was not sure if it would be used in GoQ code

Reply:
but was this added by geth and not used?

Reply:
it was removed by geth : https://github.com/ethereum/go-ethereum/pull/21579/files#diff-c08df6df90e891223dbfc3d85873d44064604480bb9b72a516facab85472aa6eL739

it was in a conflict, so I prefered to let it go in the merge, then I was sure it was not used by the linter